dfttk/workflows.py

```diff
@@ -980,8 +1053,101 @@ def elec_dos_parallel(
         with open("run_elec_dos", "w") as file:
             file.write(new_run_file)
         os.system("sbatch run_elec_dos")
-        os.chdir(path)        
-        
+        os.chdir(path)
+
+
+def charge_density_difference(
```

Comment:
Any changes to this function? or did you just move it?

Reply:
No changes.